src/containers/AssignmentTexterContact.jsx

```diff
@@ -478,7 +486,15 @@ export class AssignmentTexterContact extends React.Component {
     this.setState({ disabled: true });
     try {
       if (optOutMessageText.length) {
-        await this.props.mutations.sendMessage(message, contact.id);
+        const sendMessageResult = await this.props.mutations.sendMessage(
+          message,
+          contact.id
+        );
+        if (sendMessageResult.errors && this.props.campaign.organization.id) {
```

Comment:
I'm a little remiss to put this condition here -- especially without a specific error tested for.

Can we either test for a specific error OR can an error trigger a refresh on UserMenu or some higher component of currentUser -- where an update on *that* will test for SUSPENDED role and redirect?
We could even put it in the `hoc` super component and test for any response having a particular suspend error and then redirect there.

Basically, I don't want to need this logic in every component that tests for suspension.